packages/std/src/ibc.rs

```diff
@@ -111,7 +140,7 @@ pub enum IbcOrder {
 /// that can be compared against another Height for the purposes of updating and
 /// freezing clients.
 /// Ordering is (revision_number, timeout_height)
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[derive(Serialize, Deserialize, Copy, Clone, Debug, PartialEq, Eq, JsonSchema)]
```

Comment:
Using Copy to best effect

Reply:
This makes `Open<IbcTimeoutBlock>` `Copy` as well, which we use in the getters.